### ISSUE TITLE
Question about bulk example in README

### ISSUE THREAD

**Issue by vharmain** [mpenet/spandex#72]
Hi, thanks for the great lib! I've been using Spandex for many years with joy.

In https://github.com/mpenet/spandex#bulk-requests-scheduling example there's:

```clojure
;; setup an response consumer (we just want to make sure we don't clog this channel)
(future (loop [] (async/<!! (:output-ch c)))
```

I can't get my head around the `loop` construct here. To my understanding, `output-ch` contains the results aggregated into a single value. Also, since there's no `recur`, it'll run exactly once anyway. Is this a mistake in the example or am I missing something crucial?

Thanks!

**Comment by mpenet** [comment]
Hi,

It’s indeed a mistake in the example, it should recur as long as the chan is not closed.

**Comment by mpenet** [comment]
fixed in a118f03abb5b472141e31ac486662b02da61e17d
